tests/lava/lib/peripherals/realsense/test_realsense.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import sys
+from pathlib import Path
+
+from lava.lib.peripherals.realsense.realsense import Realsense
+from lava.magma.core.run_configs import Loihi2SimCfg
+from lava.magma.core.run_conditions import RunSteps
+
+has_valid_camera = False
+try:
+    import pyrealsense2 as rs
+
+    for device in rs.context().devices:
+        color_sensor = device.first_color_sensor()
+        depth_sensor = device.first_depth_sensor()
+
+        if color_sensor is not None and depth_sensor is not None:
+            has_valid_camera = True
+            break
+except ImportError:
+    print("Need `pyrealsense2` library installed.", file=sys.stderr)
+    exit(1)
+
+
+class TestRealsense(unittest.TestCase):
+    @unittest.skipUnless(has_valid_camera, "Requires valid Realsense camera.")
+    def test_init_camera(self) -> None:
+        """Test that the Realsense Process (reading from camera)
+        is instantiated correctly."""
+        realsense = Realsense()
+
+        if realsense.bgr_2d_shape != realsense.depth_2d_shape:
+            self.assertNotEqual(realsense.bgr_out_port.shape[:2],
+                                realsense.depth_out_port.shape)
+
+        realsense = Realsense(align_depth_to_bgr=True)
+
+        self.assertEqual(realsense.bgr_out_port.shape[:2],
+                         realsense.depth_out_port.shape)
+
+    def test_init_files(self) -> None:
+        """Test that the Realsense Process (reading from files)
+        is instantiated correctly."""
+        directory_path = Path(__file__).resolve().parent / "recording"
+
+        # TODO: Uncomment this when align_depth_to_bgr=True is supported with
```

Comment:
Are we waiting for that to happen? Otherwise, please remove this todo from the code.